x-pack/plugins/beats_management/public/components/table/assignment_schema.ts

```diff
@@ -22,20 +23,32 @@ export interface AssignmentControlSchema {
 export const beatsListAssignmentOptions: AssignmentControlSchema[] = [
   {
     grow: false,
-    name: 'Unenroll selected',
+    name: i18n.translate('xpack.beatsManagement.beatsListAssignmentOptions.unenrollButtonLabel', {
```

Comment:
somewhere you use .table namespace but here you use .beatsListAssignmentOptions. Maybe it is better to use .table namespace as well?

Reply:
No, because these are options that are displayed not on the table:

![image](https://user-images.githubusercontent.com/11524637/49153384-a063c500-f326-11e8-956e-87d757ae7c91.png)

Reply:
Ok, good